### PR TITLE
Further storage REST API tweaks.

### PR DESCRIPTION
**Pull request by mzhurovich**

### CODE REVIEW

File: Storage/api.h [hunk 1]
```diff
@@ -96,27 +96,29 @@ struct RESTfulHandlerGenerator {
...
-                                },
-                                std::move(request)).Detach();
-                  } catch (const TypeSystemParseJSONException& e) {
-                    request(handler.ErrorBadJSON(e.What()));
-                  }
-                });
+            handler.Enter(std::move(request),
+                          [&handler, &storage](Request request) {
+                            try {
+                              // `entry` is not `const` to allow modifying a copy of it inside lambda.
```
Comment (dkorolev) [inline Storage/api.h]: inside the lambda

Comment (dkorolev) [inline Storage/api.h]: or `auto mutable_entry = ...`, eliminating the need in the comment.

**Comment by dkorolev** [comment]
LGTM++

**Comment by mzhurovich** [comment]
@dkorolev Thnx, fixed.

**Comment by dkorolev** [comment]
YAY!!! :+1:

**Pull request merged**